golonomic.go

```diff
@@ -18,36 +18,52 @@ const (
 
 var inverse = mat.NewDense(3, 3, nil)
 
+var motorA *ev3dev.TachoMotor
+var motorB *ev3dev.TachoMotor
+var motorC *ev3dev.TachoMotor
+
 func setupInverse() {
 	data := []float64{
-		math.Cos(a1*math.Pi/180), math.Cos(a2*math.Pi/180), math.Cos(a3*math.Pi/180),
-		math.Sin(a1*math.Pi/180), math.Sin(a2*math.Pi/180), math.Sin(a3*math.Pi/180),
+		math.Cos(a1 * math.Pi / 180), math.Cos(a2 * math.Pi / 180), math.Cos(a3 * math.Pi / 180),
+		math.Sin(a1 * math.Pi / 180), math.Sin(a2 * math.Pi / 180), math.Sin(a3 * math.Pi / 180),
 		1, 1, 1}
 	matrix := mat.NewDense(3, 3, data)
 
 	err := inverse.Inverse(matrix)
 	if err != nil {
 		log.Fatalf("failed to inverse matrix: %v", err)
 	}
-
-	fn := mat.Formatted(inverse, mat.Prefix("    "), mat.Squeeze())
-	fmt.Printf("i = %v", fn)
 }
 
-func testMotors() {
-	outA, err := ev3dev.TachoMotorFor("ev3-ports:outA", "lego-ev3-l-motor")
+func setupMotors() {
+	var err error
+
+	motorA, err = ev3dev.TachoMotorFor("ev3-ports:outA", "lego-ev3-l-motor")
 	if err != nil {
-		log.Fatalf("failed to find medium motor on outA: %v", err)
+		log.Fatalf("failed to find large motor on outA: %v", err)
 	}
-	err = outA.SetStopAction("brake").Err()
+	err = motorA.SetStopAction("brake").Err()
 	if err != nil {
-		log.Fatalf("failed to set brake stop for medium motor on outA: %v", err)
+		log.Fatalf("failed to set brake stop for large motor on outA: %v", err)
 	}
-	maxMedium := outA.MaxSpeed()
 
-	outA.SetSpeedSetpoint(50 * maxMedium / 100).Command("run-forever")
-	time.Sleep(time.Second / 2)
-	outA.Command("stop")
+	motorB, err = ev3dev.TachoMotorFor("ev3-ports:outB", "lego-ev3-l-motor")
+	if err != nil {
+		log.Fatalf("failed to find large motor on outB: %v", err)
+	}
+	err = motorB.SetStopAction("brake").Err()
+	if err != nil {
+		log.Fatalf("failed to set brake stop for large motor on outB: %v", err)
+	}
+
+	motorC, err = ev3dev.TachoMotorFor("ev3-ports:outC", "lego-ev3-l-motor")
+	if err != nil {
+		log.Fatalf("failed to find large motor on outC: %v", err)
+	}
+	err = motorC.SetStopAction("brake").Err()
+	if err != nil {
+		log.Fatalf("failed to set brake stop for large motor on outC: %v", err)
+	}
 }
 
 func move(x, y, z float64) {
```

Comment:
i don't understand what z is in this case

Reply:
added comment and used better variable name